config/envs/rhpam-authoring-ha.yaml

```diff
@@ -290,6 +219,110 @@ others:
             - type: ConfigChange
 
     statefulsets:
+      - metadata:
+          name: "[[.ApplicationName]]-datagrid"
+          labels:
+            app: "[[.ApplicationName]]"
+            application: "[[.ApplicationName]]"
+            service: "[[.ApplicationName]]-datagrid"
+        spec:
+          podManagementPolicy: OrderedReady
+          replicas: 1
+          selector:
+            matchLabels:
+              deploymentConfig: "[[.ApplicationName]]-datagrid"
+          serviceName: "[[.ApplicationName]]-datagrid"
+          strategy:
+            rollingParams:
+              intervalSeconds: 20
+              maxSurge: 1
+              maxUnavailable: 1
+              timeoutSeconds: 1200
+              updatePeriodSeconds: 20
+            type: Rolling
+          template:
+            metadata:
+              labels:
+                app: "[[.ApplicationName]]"
+                application: "[[.ApplicationName]]"
+                deploymentConfig: "[[.ApplicationName]]-datagrid"
+              name: "[[.ApplicationName]]-datagrid"
+            spec:
+              containers:
+                - env:
+                    - name: SERVICE_NAME
+                      value: "[[.ApplicationName]]-datagrid"
+                    - name: SERVICE_PROFILE
+                      value: "[[.ApplicationName]]-datagrid"
+                    - name: JGROUPS_PING_PROTOCOL
+                      value: openshift.DNS_PING
+                    - name: OPENSHIFT_DNS_PING_SERVICE_NAME
+                      value: "[[.ApplicationName]]-datagrid-ping"
+                    - name: INFINISPAN_CONNECTORS
+                      value: "hotrod"
+                  image: "registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.1"
```

Comment:
Can we move this to constants (and replace through template params)? It now appears in 8 locations, two env files and 6 unit tests. At least with the next minor update, it can be one change instead of 8 changes?